openfisca_core/taxbenefitsystems.py

```diff
@@ -244,8 +245,24 @@ def get_column(self, column_name, check_existence = False):
     def update_column(self, column_name, new_column):
         self.column_by_name[column_name] = new_column
 
+    def neutralize_variable(self, variable_name):
+        """
+        Neutralizes an OpenFisca variable existing in the tax and benefit system.
+
+        A neutralized variable always returns its default value when computed.
+
+        Trying to set inputs for a neutralized variable has no effect.
+        """
+        self.update_column(variable_name, neutralize_column(self.get_column(variable_name)))
```

Comment:
Should this helper `neutralize_column` be renamed to `neutralized_variable` (I insist on *neutralized*, like in `new_filled_column`)?

Because `TaxBenefitSystem.neutralize_variable` is an action, whereas the helper in `formulas.py` is a function which creates a new `Variable`.